src/sa_web/static/js/views/map-view.js

```diff
@@ -19,14 +19,13 @@ var Shareabouts = Shareabouts || {};
 
       self.map = L.map(self.el, self.options.mapConfig.options);
 
+
```

Comment:
Let's be sure to clean out all of our commented-out code, blank lines, etc before merging